tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -116,6 +116,7 @@ public TracerSettings(IConfigurationSource? source)
             var headerTagsNormalizationFixEnabled = source.GetBool(ConfigurationKeys.FeatureFlags.HeaderTagsNormalizationFixEnabled) ?? true;
             // Filter out tags with empty keys or empty values, and trim whitespaces
             HeaderTags = InitializeHeaderTags(inputHeaderTags, headerTagsNormalizationFixEnabled);
+            MetadataSchemaVersion = source.GetString(ConfigurationKeys.MetadataSchemaVersion) ?? "v0";
```

Comment:
Do we need to validate the schema version is the correct value i.e. it's `v1` not `1`, `version1`, `V1` etc?

Reply:
I had placed the validation inside the ServiceNames constructor, but I think you're right that it makes sense to place here. Will update